test/backend-cpp/CMakeLists.txt

```diff
@@ -45,10 +45,9 @@ function(add_backend_cpp_unittest test_name)
   endif()
 endfunction()
 
-set(TEST_LINK_LIBS CompilerUtils ExecutionSession)
+set(TEST_LINK_LIBS ModelLib)
```

Comment:
You could just reference the lib itself below since there's just the one left.

Reply:
done